src/System.Runtime.Numerics/tests/ComplexTests.cs

```diff
@@ -1218,6 +1291,34 @@ public static void Multiply(double realLeft, double imaginaryLeft, double realRi
             VerifyRealImaginaryProperties(result, expectedReal, expectedImaginary);
         }
 
+        [Theory]
+        [MemberData(nameof(Multiply_TestData))]
+        [MemberData(nameof(SmallRandom_4_TestData))]
+        //[MemberData(nameof(Invalid_4_TestData))]
```

Comment:
Why is this is commented out? If it is needed to be commented out, there should be a comment/reason and possibly an associated issue as well.

Reply:
@eerhardt and @tannergooding This is slipped when I was busy with division operations. Replicate "non-value changing" optimization approach for multiplication operations and uncomment test cases.